backend/src/main/java/ai/verta/modeldb/experiment/ExperimentDAORdbImpl.java

```diff
@@ -528,31 +531,6 @@ public Experiment deleteExperimentAttributes(
     }
   }
 
-  @Override
-  public Boolean deleteExperiment(String experimentId) {
```

Comment:
Where did this go?

Reply:
Not in use so removed it. we have another function called `deleteExperiments` to delete entities.

Reply:
Make a separate PR for it. Let's not shove unrelated changes together for no reason.

Reply:
reverted